projectroles/tests/test_tasks.py

```diff
@@ -0,0 +1,115 @@
+from test_plus.test import TestCase
```

Comment:
Please add a comment to the start of the module, see other test modules for examples.